src/components/TodoList.tsx

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+import { Todo } from '../types/Todo';
+import cn from 'classnames';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
+      {todos.map(todo => (
```

Comment:
You can make one more component, like we did before in our projects `{todos.map(todo => (
<OneTodo key={} todo={todo}>
)}`